README.md

```diff
@@ -52,42 +51,52 @@ have a permissive auto-approval on all Dependabot PRs like so:
 
 ```yaml
 name: Dependabot auto-approve
-description: Auto-approve Dependabot PRs
 on: pull_request_target
 permissions:
   pull-requests: write
 jobs:
   dependabot:
+    runs-on: ubuntu-latest
     # Checking the actor will prevent your Action run failing on non-Dependabot PRs
     if: ${{ github.actor == 'dependabot[bot]' }}
     steps:
       - name: Dependabot metadata
         id: metadata
-        uses: dependabot/fetch-metadata
+        uses: dependabot/fetch-metadata@v1.1.0
```

Comment:
I've set this ahead to v1.1.0 so we can use that version as the first 'release' version while still leaving room for some more patches while we test.